script/generate-octicons.ts

```diff
@@ -9,11 +7,11 @@
 
 'use strict'
 
-const fs = require('fs')
-const process = require('process')
-const xml2js = require('xml2js')
-const path = require('path')
-const toCamelCase = require('to-camel-case')
+import fs = require('fs')
+import process = require('process')
```

Comment:
And this can be removed - `process` is a global and only used for `process.exit` here.